src/review-list/dialog/review-item.scss

```diff
@@ -17,18 +17,18 @@
         font-family: Arial, sans-serif;
         border-collapse: collapse;
         width: 100%;
-    }
 
-    td,
-    th {
-        border: 1px solid #000;
-        text-align: left;
-        font-size: 8px;
-        padding: 8px;
-        width: 80px;
-    }
+        td,
+        th {
+            border: 1px solid #000;
+            text-align: left;
+            font-size: 8px;
+            padding: 8px;
+            width: 80px;
+        }
```

Comment:
Format this file to use 2 spaces for tabs

Reply:
do we have a global prettier config that we can add or i have to go through this manually